### ISSUE TITLE
Does Grounding DINO support training

### ISSUE THREAD

**Issue by Josh1021** [open-mmlab/mmdetection#11263]
I compared grounding dino in mmdet and [Open-grounding-dino](https://github.com/longzw1997/Open-GroundingDino), and the loss of two codes are same, so can i  train G-dino from scratch use mmdet(not finetune, because finetune will loss the ability of zero-shot)?

**Comment by hhaAndroid** [comment]
@Josh1021 The current version of this algorithm only supports fine-tuning and does not support pre-training. However, we will release a stronger version by the end of this month that supports both pre-training and fine-tuning.

**Comment by Josh1021** [comment]
@hhaAndroid   OK，Thank you for your response and great work！And can you give me a hint about why training is not supported now in mmdet? From my understanding, if it supports fine-tuning, it should allow training, right?
